src/azure-cli/azure/cli/command_modules/servicebus/_params.py

```diff
@@ -105,11 +105,11 @@ def load_arguments_sb(self, _):
 
     with self.argument_context('servicebus queue update') as c:
         c.argument('enable_partitioning', arg_type=get_three_state_flag(),
-                   help='A boolean value that indicates whether the queue is to be partitioned across multiple message brokers.', deprecate_info=c.deprecate(hide=True, expiration='2.45.0'))
-        c.argument('requires_session', options_list=['--enable-session'], arg_type=get_three_state_flag(), help='A boolean value indicating whether the queue supports the concept of sessions.', deprecate_info=c.deprecate(hide=True, expiration='2.45.0'))
+                   help='A boolean value that indicates whether the queue is to be partitioned across multiple message brokers.', deprecate_info=c.deprecate(hide=True, expiration='2.49.0'))
+        c.argument('requires_session', options_list=['--enable-session'], arg_type=get_three_state_flag(), help='A boolean value indicating whether the queue supports the concept of sessions.', deprecate_info=c.deprecate(hide=True, expiration='2.49.0'))
```

Comment:
Same issue happened multiple times before, such as https://github.com/Azure/azure-cli/pull/21892

Maybe we can remove totally `expiration`, otherwise it can trigger failure again in 2.49.0 release.